unify_emphasis.py

```diff
@@ -0,0 +1,163 @@
+    "ManagingHostsDocURL": "ManagingHostsDocTitle",
+    "ManagingOrganizationsLocationsDocURL": "ManagingOrganizationsLocationsDocTitle",
+    "PlanningDocURL": "PlanningDocTitle",
+    "ProvisioningDocURL": "ProvisioningDocTitle",
+    "UpgradingDocURL": "UpgradingDocTitle",
+    "InstallingDisconnectedDocURL": "InstallingDisconnectedDocTitle",  # FIXME: Add to attributes?
+}
+list_of_adoc_files = []
+pattern_docurl = r"([Ss])*(ee )*(in )*(the )*({)([A-Za-z]*DocURL)(})([A-Za-z_-]*)(\[)([_]*)([A-Z{}a-z0-9 ()-]*)([_]*)(\])*( )*(in )*(the )*(_)*([A-Za-z{} ]*)(_ )*([Gg]uide)*(_)*"
+
+# FIXME: These titles have no DocURL?
+"""
+list of DocTitles:
+    * AdministeringAnsibleDocTitle
+    * AppCentricDeploymentDocTitle
+    * ConvertingHostRHELDocTitle
+    * DeployingAWSDocTitle
+    * InstallingServerDisconnectedDocTitle
+    * QuickstartDocTitle
+    * ReleaseNotesDocTitle
+"""
+
+result = subprocess.run(["git", "restore", "guides/"], stdout=subprocess.PIPE)
+print(result.stdout.decode("utf-8"))
+
+# https://stackoverflow.com/a/3964691
+for root, dirs, files in os.walk("./guides"):
+    for file in files:
+        if file.endswith(".adoc"):
+            list_of_adoc_files.append(os.path.join(root, file))
+
+for adoc_file in list_of_adoc_files:
+    with open(str(adoc_file), "r") as f:
+        file_content = f.read().splitlines(keepends=True)
+        new_content = []
+        for line in file_content:
+            matches = re.finditer(pattern_docurl, line)
+            for match in matches:
+                mg1 = match.group(1)  # optional: capitalization of 's' of '[Ss]ee'; necessary if sentence starts with 'See'
+                mg2 = match.group(2)  # optional: 'ee ' of '[Ss]ee'
+                mg3 = match.group(3)  # optional: 'in '
+                mg4 = match.group(4)  # optional: 'the '
+                mg5 = match.group(5)  # opening curly bracket
+                mg6 = match.group(6)  # DocURL
+                mg7 = match.group(7)  # closing curly bracket
+                mg8 = match.group(8)  # optional: anchor
+                mg9 = match.group(9)  # opening bracket
+                mg10 = match.group(10)  # optional: '_' to make title italic
+                mg11 = match.group(11)  # title
+                mg12 = match.group(12)  # optional: '_' to make title italic
+                mg13 = match.group(13)  # closing bracket
+                mg14 = match.group(14)  # optional: ' ' if sentence continues.
+                mg15 = match.group(15)  # optional: 'in '
+                mg16 = match.group(16)  # optional: 'the '
+                mg17 = match.group(17)  # optional: '_' to make DocURL italic
+                mg18 = match.group(18)  # optional: DocURL title
+                mg19 = match.group(19)  # optional: '_' to make DocURL italic
+                mg20 = match.group(20)  # optional: '[Gg]uide'
+                mg21 = match.group(21)  # optional: '_' to make [Gg] italic
```

Comment:
I'd have a look at [named capture groups](https://docs.python.org/3/howto/regex.html#non-capturing-and-named-groups) because numbers get mixed up too easily.